app/controllers/application_controller.rb

```diff
@@ -46,10 +50,10 @@ def authentication_failed
     end
   end
 
-  def resource_not_found_handler(exception)
+  def generic_exception_handler(exception)
     respond_to do |format|
       format.json do
-        render json: {status: 404, reason: exception.to_s}, status: 404
+        render json: {status: exception.status, reason: exception.to_s}, status: exception.status
```

Comment:
Tutaj powinno być więc zawsze status: "error"

A faktycznie to było wcześniej - nie zauważyłem.